docs/testing.rst

```diff
@@ -82,6 +82,23 @@ To download geckodriver and have it ready to run in your system, there are a cou
 
             brew install geckodriver
 
+To download chromedriver and have it ready to run in your system, there are a couple of ways:
```

Comment:
This all looks good, but it's in the functional tests section of the document, which comes after the Jasmine instructions. Sorry if I wasn't clear, but I was suggesting we could put this information at the top of the file, as a requirement for running both Jasmine and functional tests, so that people would read it before getting to the Jasmine instructions. Does that make sense?

Reply:
yup, my mistake!